content/_data/changelogs/weekly.yml

```diff
@@ -9391,6 +9391,119 @@
+    Refer to the <a href="/blog/2020/11/10/spring-xstream/">Spring and XStream updates (breaking changes!)</a> blog post for more details.
+  changes:
+  - type: major bug
+    category: major bug
+    pull: 5046
+    issue: 64040
+    authors:
+    - timja
+    message: |-
+      Fix hidden page elements from radio blocks appearing when they should not.
+  - type: bug
+    category: bug
+    pull: 5043
+    authors:
+    - jeffret-b
+    references:
+      - pull: 5043
+      - url: "https://github.com/jenkinsci/remoting/releases/tag/remoting-4.6"
+        title: Remoting 4.6 changelog
+    message: |-
+      Upgrade from Remoting 4.5 to Remoting 4.6 with bugfixes and dependency updates.
+  - type: rfe
+    category: rfe
+    pull: 5008
+    issue: 62446
+    authors:
+    - aHenryJard
+    message: |-
+      Make the Maven Configuration of Global Tools compatible with configuration as code plugin.
+  - type: rfe
+    category: rfe
+    pull: 4294
+    issue: 41891
+    authors:
+    - jsoref
+    message: |-
+      Minor English cleanup for resource root URL setting.
+  - type: major rfe
+    category: developer
+    pull: 4944
+    issue: 19561
+    authors:
+    - jglick
+    references:
+      - pull: 4944
+      - url: "https://github.com/jenkinsci/jep/blob/master/jep/227/README.adoc"
+        title: JEP-227
+      - url: "/blog/2020/11/10/spring-xstream/"
+        title: Spring and XStream updates (breaking changes!) blog post
+    message: |-
+      Developer: Jenkins now uses an updated version of the XStream serialization library without custom patches.
+  - type: major rfe
+    category: developer
+    pull: 4848
+    authors:
+    - jglick
+    references:
+      - pull: 4948
+      - url: "https://github.com/jenkinsci/jep/blob/master/jep/228/README.adoc"
+        title: JEP-228
+      - url: "/blog/2020/11/10/spring-xstream/"
+        title: Spring and XStream updates (breaking changes!) blog post
+    message: |-
+      Developer: Jenkins now uses Spring Security rather than its predecessor, Acegi Security.
+      Other bundled Spring libraries are also updated.
+  - type: major rfe
+    category: developer
+    pull: 4929
+    authors:
+    - fqueiruga
+    references:
+      - pull: 4929
+      - url: "https://blog.jquery.com/"
+        title: jQuery older release notes (3.1.1, 3.2.1, 3.3.0, 3.3.1, 3.4.0, 3.4.1)
+      - url: "https://blog.jquery.com/2020/04/10/jquery-3-5-0-released/"
+        title: jQuery 3.5.0 release notes
+      - url: "https://blog.jquery.com/2020/05/04/jquery-3-5-1-released-fixing-a-regression/"
+        title: jQuery 3.5.1 release notes
+    message: |-
+      Developer: Upgrade jQuery from 2.1.4 to 3.5.1.
+  - type: rfe
+    category: developer
+    pull: 5042
+    authors:
+    - jglick
+    message: |-
+      Developer: Removing the deprecated <code>zeroclipboard</code> UI component and the unused dependency on ZeroClipboard.
```

Comment:
```suggestion
      Developer: Remove the deprecated <code>zeroclipboard</code> UI component and the unused dependency on ZeroClipboard.
```

Reply:
Thanks for catching that.  Fixed in 72347439123c5c2c8c3cb70443555e5b1751fca5